src/components/Acknowledgement/Acknowledgement.tsx

```diff
@@ -179,7 +179,7 @@ export const Acknowledgement = ({
               />
               <Box
                 textAlign={'center'}
-                maxWidth='90%'
+                maxWidth='100%'
```

Comment:
This does make the content in only other consumer of this component with the `content` prop, `ArbitrumBridgeAcknowledgement` a bit wider:

<img width="349" alt="Screenshot 2024-07-05 at 5 34 06 PM" src="https://github.com/shapeshift/web/assets/97164662/0c878774-ad9e-47eb-9ae1-3ac089198724">

Reply:
I don't think we'll want to prod drill this - ideally all of our acknowledgement's look and feel similar style-wise.